auto_route/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: auto_route
 description: AutoRoute is a declarative routing solution, where everything needed for navigation is automatically generated for you.
-version: 10.0.1
+version: 10.1.1
```

Comment:
I raised the version to 10.1.1 and updated the changelog.md file.

Let me know if you wish to have another version